noetic/distribution.yaml

```diff
@@ -3020,6 +3020,21 @@ repositories:
       url: https://github.com/at-wat/hokuyo3d.git
       version: master
     status: developed
+  human_description:
+    doc:
+      type: git
+      url: https://github.com/ros4hri/human_description.git
+      version: 1.0.0
```

Comment:
The version here should point to a branch (the development branch used to produce `1.0.0`, e.g. `main`).
Same for the version under the `source` entry.